src/common/io/src/format_settings.rs

```diff
@@ -13,145 +13,17 @@
 // limitations under the License.
 
 use chrono_tz::Tz;
-use common_exception::ErrorCode;
-use common_exception::Result;
-
-use crate::consts::*;
-
-// fixed the format in struct/array,
-// when it`s repr as a string in csv/tsv/json/...
-// should be compatible with the format used SQL
-#[derive(Debug, Clone, PartialEq, Eq)]
-pub struct NestedFormatSettings {
-    pub true_bytes: Vec<u8>,
-    pub false_bytes: Vec<u8>,
-    pub null_bytes: Vec<u8>,
-    pub nan_bytes: Vec<u8>,
-    pub inf_bytes: Vec<u8>,
-    pub quote_char: u8,
-}
-
-impl Default for NestedFormatSettings {
-    fn default() -> Self {
-        NestedFormatSettings {
-            true_bytes: TRUE_BYTES_NUM.as_bytes().to_vec(),
-            false_bytes: FALSE_BYTES_NUM.as_bytes().to_vec(),
-            null_bytes: NULL_BYTES_UPPER.as_bytes().to_vec(),
-            nan_bytes: NAN_BYTES_LOWER.as_bytes().to_vec(),
-            inf_bytes: INF_BYTES_LOWER.as_bytes().to_vec(),
-            quote_char: b'\'',
-        }
-    }
-}
 
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct FormatSettings {
-    // both
     pub timezone: Tz,
```

Comment:
This strict only has one field timezone? Maybe we can extract it, and remove this strict?

Reply:
Will be refactor after new expression merged.

Reply:
almost remove the usage of FormatSetting, leave alone the usage in Variant, which will be remove after new expression is used https://github.com/datafuselabs/databend/issues/8949